src/go/pkg/controller/approllout/controller_test.go

```diff
@@ -319,6 +321,174 @@ spec:
 	}
 }
 
+// generateApp will generate an app for testing
+func generateApp(name, version, robotPayload, cloudPayload string) apps.App {
+	app := apps.App{}
```

Comment:
can you write:
app := apps.App{
  Name: name,
  Labels: map[string]string{
    ..
  },
}

}

Reply:
thanks, good catch!